src/Tools/ExternalAccess/Razor/Remote/RazorRemoteServiceConnectionWrapper.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Remote;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.Razor
+{
+    internal readonly struct RazorRemoteServiceConnectionWrapper<TService> : IDisposable
```

Comment:
❔ Why are we making a wrapper if it's not part of the RIVT API?

Reply:
This is not RIVT but a separate assembly.